tf_fastmri_data/h5.py

```diff
@@ -50,8 +50,11 @@ def load_data_from_file(fpath, slice_random=False, no_kspace=False, multicoil=Fa
     else:
         slices = (0, n_slices)
     if mode == 'train':
-        image = h5_tensors(image_name)[slices[0]:slices[1]]
-        if slice_random:
+        if slice_num > -1:
+            image = h5_tensors(image_name)[slice_num:slice_num+1]
```

Comment:
I think you are better off using another condition when defining `slices`. Like:
```python
elif slice_index >-1:
	slices = (slice_index, slice_index + 1)
```

This way the rest of the implem can remain unchanged.

Reply:
Ahh yes..